redash/handlers/admin.py

```diff
@@ -41,21 +41,6 @@ def outdated_queries():
     return json_response(response)
 
 
-@routes.route("/api/admin/queries/tasks", methods=["GET"])
-@require_super_admin
-@login_required
-def queries_tasks():
-    record_event(
-        current_org,
-        current_user._get_current_object(),
-        {"action": "list", "object_type": "celery_tasks"},
-    )
-
-    response = {"tasks": celery_tasks()}
-
-    return json_response(response)
-
-
 @routes.route("/api/admin/queries/rq_status", methods=["GET"])
```

Comment:
Maybe we should switch the URLs not to leak the implementation detail (RQ) now that it's only RQ?